src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -115,6 +115,7 @@
     public static final String GIT_COMMIT = "GIT_COMMIT";
     public static final String GIT_PREVIOUS_COMMIT = "GIT_PREVIOUS_COMMIT";
     public static final String GIT_PREVIOUS_SUCCESSFUL_COMMIT = "GIT_PREVIOUS_SUCCESSFUL_COMMIT";
+    public static final boolean DEFAULT_HIDE_EXCLUDED_COMMITS_IN_CHANGESET = false;
```

Comment:
@ndeloof, do you mean system properties, set by -Dprop=value ?
Do you want the value to be initialized in static constructor, then?